Wire-iOS/Sources/Analytics/Events/Analytics+CallStateEvents.swift

```diff
@@ -51,6 +84,27 @@ extension AnalyticsEvent {
         return event
     }
 
+    static func endedCall(asVideoCall: Bool,
+                          callDirection: CallDirection,
+                          callDuration: Double,
+                          callParticipants: Double,
```

Comment:
```suggestion
                          callParticipants: Int,
```

I think an Int is more suitable here.